data/ui/oops-menus.ui

```diff
@@ -17,11 +17,11 @@
         <attribute name="action">win.details</attribute>
       </item>
       <item>
-        <attribute name="label" translatable="yes">_Open problem's data directory</attribute>
+        <attribute name="label" translatable="yes">_Open problem’s data directory</attribute>
         <attribute name="action">win.open-directory</attribute>
       </item>
       <item>
-        <attribute name="label" translatable="yes">_Copy problem's ID to Clipboard</attribute>
+        <attribute name="label" translatable="yes">_Copy problem’s ID to Clipboard</attribute>
```

Comment:
To be honest, the possessive feels weird here altogether (and the capitalization of “clipboard”). “Copy problem ID to clipboard” sounds way better.

Reply:
(Same with “problem’s data” above)